Exercise_A/index.html

```diff
@@ -4,14 +4,95 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+      integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+      crossorigin="anonymous"
+   />
+
+    <script
+      src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+      integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+      crossorigin="anonymous"
+    ></script>
+
+
+    <script
+      src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
+      integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
+      crossorigin="anonymous"
+    ></script>
+
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
+      integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
+      crossorigin="anonymous"
+    ></script>
+
+    <title>Exercise: Create a simple webpage</title>
     <meta name="description" content="" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1"
     />
+
+  <link rel="stylesheet" href="index.css">
   </head>
+  <header>
+    <!--Navbar-->
+    <nav class="navbar navbar-expand-lg navbar-light bg-white">
+        <a class="navbar-brand" href="#">
+          <img src="./assets/bootstrap.png" alt="logo" style="height: 50px">
+        </a>
+        <ul class="navbar-nav">
+          <li class="nav-item active">
+            <a class="nav-link" href="#">Home</a>
+          </li>
+          <li class="nav-item active">
+            <a class="nav-link" href="#">Link</a>
+          </li>
+        </ul>
+    </nav>
+    <!--Navbar-->
+  </header>
   <body>
-    <!-- CODE HERE -->
+    <!--List-->
+    <div>
+      <ul class="list-group">
+        <li class="list-group-item">Item I</li>
+        <li class="list-group-item">Item II</li>
+        <li class="list-group-item">Item III</li>
+        <li class="list-group-item">Item IV</li>
+        <li class="list-group-item">Item V</li>
+      </ul>
+    </div>
+    <!--List-->
+    <!--Buttons-->
+    <div class="row">
+      <div class="col">
+        <button
+          type="button"
+          class="btn btn-outline-primary"
+        >Primary
+        </button>
+        <button
+          type="button"
+          class="btn btn-outline-secondary"
+        >Second
+        </button>
+        <button
+          type="button"
+          class="btn btn-outline-success"
+          >Success
+        </button>
+        <button
+          type="button"
+          class="btn btn-outline-danger"
+          >Danger
+        </button>
+      </div>
+    </div>
+
   </body>

```

Comment:
Great use of comments and fantastic implementation of bootstrap to style the components. Nice work!